src/ToDo/Business/Services/IAuthenticationService.cs

```diff
@@ -0,0 +1,11 @@
+﻿using ToDo.Business.Entities;
+
+namespace ToDo.Business.Services
+{
+	public interface IAuthenticationService
+	{
+		//This will be called from WelcomeViewModel in order to get access token and user's data(email,name).
+		//because if the user tap on Close icon on the webview authentication screen the user should be redirected to WelcomePage
+		Task<string> LoginAsync();
```

Comment:
The returned value is a token?  I would expect a more complex return value than just a string... There's a lot of nice stuff in the [`AuthenticationResult`](https://docs.microsoft.com/en-us/dotnet/api/microsoft.identity.client.authenticationresult?view=azure-dotnet) that may be useful to the application... no?  Don't forget we're building a reference implementation application, so we want clean, self-described detailed interfaces.

Reply:
I modified the return type to UserContext(Email, Name and AccessToken). We will need it in the HomeViewModel in order to show user's data like Email and Name...